server/jobs/offboarding/email.js

```diff
@@ -9,6 +10,9 @@ import { executeEmailAutomation } from '../helpers';
  * @param {object} automationResult Result of automation job
  * @returns {void}
  */
-export default function emailOffboarding(placement) {
-  executeEmailAutomation([sendSOPOffboardingMail, sendITOffboardingMail], placement);
-}
+const emailOffboarding = async (placement) => {
+  placement = await fetchNewPlacements('External Engagements - Rolling Off', 1);
```

Comment:
Nice job. However I think `fetchNewPlacements` function no longer takes two arguments from the fix being done in replacing mock-api post endpoints with seeder